config/layout.scss

```diff
@@ -8,3 +8,7 @@ $layout-width: $layout-content-width + $layout-sidebar-width;
 
 $layout-width-m: 660px;
 $layout-width-l: 1280px;
+
+$layout-section-spacing-l: 15px;
```

Comment:
Don't really like the naming - suggestions?

Reply:
$offset-s/m/l

Reply:
-> offset.scss

Reply:
you can use sizes.scss there is $offset-space already